src/chrono_granular/physics/ChGranular.cpp

```diff
@@ -172,10 +172,11 @@ void ChSystemGranularSMC::packSphereDataPointers() {
         sphere_data->contact_history_map = contact_history_map.data();
     }
 
-    // force prefetch the sphere data pointer after update
-    int dev_ID;
-    gpuErrchk(cudaGetDevice(&dev_ID));
-    gpuErrchk(cudaMemPrefetchAsync(sphere_data, sizeof(*sphere_data), dev_ID));
+    // DN: had to comment this prefetch for now; crashing on Windows
+    //// force prefetch the sphere data pointer after update:
+    // int dev_ID;
+    // gpuErrchk(cudaGetDevice(&dev_ID));
+    // gpuErrchk(cudaMemPrefetchAsync(sphere_data, sizeof(*sphere_data), dev_ID));
```

Comment:
This is a concern if we care about multi-GPU

Reply:
Nic - let's focus on validation for now and not worry for the time being about the multi-GPU.